release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2884,6 +2884,46 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.17.27
+[id="ocp-4-17-27_{context}"]
+=== RHSA-2025:4204 - {product-title} {product-version}.27 bug fix and security update advisory
+
+Issued: 30 April 2025
+
+{product-title} release {product-version}.27 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:4204[RHSA-2025:4204] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:4206[RHBA-2025:4206] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.17.27 --pullspecs
+----
+
+[id="ocp-4-17-27-known-issues_{context}"]
+==== Known issues
+
+* There is a known issue when the grandmaster clock (T-GM) transitions to the Locked state too soon. This happens before the Digital Phase-Locked Loop (DPLL) completes its transition to the `Locked-HO-Acquired` state, and after the Global Navigation Satellite Systems (GNSS) time source is restored.
```

Comment:
```suggestion
* There is a known issue when the grandmaster clock (T-GM) transitions to the Locked state too soon. This happens before the Digital Phase-Locked Loop (DPLL) completes its transition to the `Locked-HO-Acquired` state, and after the Global Navigation Satellite Systems (GNSS) time source is restored. (link:https://issues.redhat.com/browse/OCPBUGS-54534[*OCPBUGS-54534*])
```

Reply:
Just this final comment and you can then add a merge-label to this PR. Thanks, Ted! Nice work.